### ISSUE TITLE
RTL8723AE does not show any WiFi networks on debian jessie

### ISSUE THREAD

**Issue by mrglobule** [lwfinger/rtlwifi_new#48]
Goodmorning,
i have a trouble with the drivers rtlwifi_new on debian 8 jessie x64.
I can install thoses drivers but when i try in command:
iwlist wlan0 scan
I a have
no scan result

can you help me?
Regards.

**Comment by lwfinger** [comment]
I'm not sure. It works for me.

There are several branches in that get repo. Have you tried any of the others? As you did not specify a branch, you are probably using master.

In addition, please list the results of 'rfkill list".

**Comment by mrglobule** [comment]
Hello, thanks for reply.
I try with master and with troy, but two are the same result.

here are the output of rfkill list:

0: hci0: Bluetooth
        Soft blocked: no
        Hard blocked: no
1: phy0: Wireless LAN
        Soft blocked: no
        Hard blocked: no

**Comment by lwfinger** [comment]
Please post the actual output from "iwscan list".

**Comment by mrglobule** [comment]
# iwlist scan
eth0      Interface doesn't support scanning.

wlan0     No scan results

usb0      Interface doesn't support scanning.

lo        Interface doesn't support scanning.

**Comment by lambertdw** [comment]
Sorry, I threw in the towel an bought the Intel wireless card.
Lenovo advertised the 8192 as "the special Lenovo wireless 2x2 radio 
that pulls in signals from miles around." rather than "chip with nothing 
but bad reviews."
Thank you.
Dave Lambert

**Comment by mrglobule** [comment]
@lwfinger
I try with debian sid, and with jessie with kernel 4, and i have the same problem...

**Comment by misac1987** [comment]
I have got same problem

**Comment by lwfinger** [comment]
You should try the following:

su -
echo "msi=1" > /etc/modprobe.d/50-rtl8723ae.conf
modprobe -rv rtl8723ae
modprobe -v rtl8723ae

If this works, it is because your motherboard does not support normal interrupts the way that most do. It will only be necessary to do this once, BUT you must be running code from this repo. A fix that would allow this fix to work with mainline or stable kernels has been submitted, but it will take a while for it to propagate though the official kernel sources.

**Comment by misac1987** [comment]
Thank you "msi=1" fix my problem.

**Comment by AA1978** [comment]
Dear lwfinger, I  I have successfully installed your driver for RTL8723AE wireless card on MSI S12 notebook. But I am experiencing nearly the same problem: "No scan results". When I try "msi=1" option, and "run modprobe -rv rtl8723ae", I get the following output:

libkmod: ERROR ../libkmod/libkmod-config.c:635 kmod_config_parse: /etc/modprobe.d/50-rtl8723ae.conf line 1: ignoring bad line starting with 'msi=1'
rmmod rtl8723ae
rmmod rtl_pci
rmmod rtlwifi
rmmod mac80211
rmmod cfg80211
rmmod rtl8723_common
rmmod btcoexist

Can you help me?

**Comment by lwfinger** [comment]
The error says that the options file is wrong and preventing the driver from loading. You need to fix that!

Are you really sure you need to change the "msi" parameter>

**Comment by AA1978** [comment]
Thank you for your reply. No, I am not sure I need this, I just tried. What else can I do to make my wireless card work properly?

**Comment by lwfinger** [comment]
You might show any errors that show up once you fix that faulty file in /etc/modprobe.d/.

**Comment by AA1978** [comment]
The error that I see is "No scan result" when I try to search for wireless networks with iwlist command. May be I should look for errors in log files? (The same wireless card works perfectly in Windows.)

**Comment by lwfinger** [comment]
It is probably blocked by rfkill. What does "rfkill list" show?

**Comment by AA1978** [comment]
No, nothing is blocked:
0: hci0: Bluetooth
        Soft blocked: no
        Hard blocked: no
1: phy0: Wireless LAN
        Soft blocked: no
        Hard blocked: no

**Comment by lwfinger** [comment]
A completely new version of the drivers has just been pushed. It fixes a lot of things. As a result, I am closing all open issues. If you get a new one, please create a new issue.

Note: The new driver fixes the problems with power save. It is recommended that you clear all ips, swlps, and fwlps options.
